src/App.tsx

```diff
@@ -1,10 +1,30 @@
+import {
+  Route,
+  RouterProvider,
+  createBrowserRouter,
+  createRoutesFromElements,
+} from 'react-router-dom';
 import './App.css';
-import QuizResults from './components/UI/quiz_results/QuizResults';
+import QuizResults from './pages/QuizResults';
+import QuizSettings from './pages/QuizSettings';
+import MainQuizScreen from './pages/MainQuizScreen';
+import Statistics from './pages/Statistics';
+
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <>
+      <Route path="/" element={<QuizSettings />} />
+      <Route path="main" element={<MainQuizScreen />} />
+      <Route path="results" element={<QuizResults />} />
+      <Route path="stats" element={<Statistics />} />
```

Comment:
I highly recommend you to create an object where you'll enumerate all paths
```ts
const paths = {
  home: '/'
  main: '/main'
  //...
}
// 
<Route path={paths.home} element={<QuizSettings />} />
//
router.navigate(paths.home)
```
this is more safe if you decide to change paths in the future